spec/system/submission_details_spec.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe "A11y", :js do
+  describe "Logged-in as a Challenge Manager" do
+    let(:user) { create_user(role: "challenge_manager") }
+    let(:challenge) { create_challenge(user: user, title: "Boston Tea Party Cleanup") }
+    let(:submission) { create(:submission, manager: user, challenge: challenge) }
+
+
+    before { system_login_user(user) }
+
+    it "submission details page is accessible" do
+      visit submission_path(submission)
+      expect(user.role).to eq("challenge_manager")
+      expect(page).to have_content(submission.id)
+      expect(page).to(be_axe_clean)
+    end
+
+    it "allows manipulation of judging status" do
+      visit submission_path(submission)
+
+      find('#eligible-for-evaluation').click
+      updated_submission = Submission.find(submission.id)
+      expect(updated_submission.judging_status).to eq('selected')
+
+      find('#selected-to-advance').click
```

Comment:
my linter suggests this locally for [Capybara/SpecificFinders](https://docs.rubocop.org/rubocop-capybara/cops_capybara.html#capybaraspecificfinders)
```suggestion
      find_by_id('selected-to-advance').click
```